Lesson11/merve/index.html

```diff
@@ -0,0 +1,46 @@
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <nav id="navbar" class="nav">
+  <a href class="a--nav-selector" id="profile-link" target="_blank"> ABOUT</a>
+  <a href="project-section" class="a--nav-selector">WORK</a>
+  <a href="contact-section" class="a--nav-selector">CONTACT</a>
+  </nav>
+  <body>
```

Comment:
All visible elements should be inside <body> element, make sure to move <nav> element here.